photonlib-java-examples/aimandrange/vendordeps/photonlib.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
ok actually this time. Since this is expected to change every release, should it be archived? churning it over in my head.... it would be needed to keep the example complete, but expected to change every release.... I think that's ok?

Reply:
I would vote add to gitignore?